src/main/java/com/johan/blignaut/entelect/vitalityincubator/task2/service/GeoDBMappingServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.johan.blignaut.entelect.vitalityincubator.task2.service;
+
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.CityClient;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.GeoDBClient;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.LinkClient;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.MetaDataClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.CityServer;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.GeoDBServer;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.LinkServer;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.MetaDataServer;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class GeoDBMappingServiceImpl implements GeoDBMappingService {
+    @Override
+    public GeoDBServer mapClientToServerGeoDB(GeoDBClient geoDBClient) {
```

Comment:
A bit of a nitpick: Since this is the only method in this class called outside of this class, it is the only one that needs to be public.
The other methods could be removed from the interface.

Reply:
Should I remove them and combine them in that method, or should I rather make them private, so as to keep them single responsibility-like?

Reply:
Oh wait, methods in an interface can't be private, my bad. Will remove them.